cloudflare/src/endpoints/account/mod.rs

```diff
@@ -15,9 +15,9 @@ pub struct Account {
     /// Account name
     pub name: String,
     /// Account Settings
-    pub settings: Settings,
+    pub settings: Option<Settings>,
     /// describes when the account was created
-    pub created_on: DateTime<Utc>,
+    pub created_on: Option<DateTime<Utc>>,
```

Comment:
How can the `created_on` field be null?

Reply:
It is null if no `created_on` field is returned in the response.

Reply:
I've stumbled with this case. `account` object only has `id` and `name`. `settings` is also missing which causes an error.

Here's a response from my CF account:

```json
{
    "result": [
        {
            "id": "*************",
            "name": "**********************",
            "status": "active",
            "paused": false,
            "type": "full",
            "development_mode": 0,
            "name_servers": [
                "jamie.ns.cloudflare.com",
                "miles.ns.cloudflare.com"
            ],
            "original_name_servers": [
                ....
            ],
            "original_registrar": null,
            "original_dnshost": null,
            "modified_on": "2020-02-20T14:35:58.750653Z",
            "created_on": "2018-04-15T15:11:48.561098Z",
            "activated_on": "2018-04-15T18:41:19.547134Z",
            "meta": {
                "step": 4,
                "wildcard_proxiable": false,
                "custom_certificate_quota": 0,
                "page_rule_quota": 3,
                "phishing_detected": false,
                "multiple_railguns_allowed": false
            },
            "owner": {
                "id": "*****************",
                "type": "user",
                "email": "********@gmail.com"
            },
            "account": {
                "id": "*************5ab6d6697377b30376",
                "name": "******@gmail.com"
            },
            "permissions": [
                "#access:edit",
                "#access:read",
                "#analytics:read",
                "#app:edit",
                "#auditlogs:read",
                "#billing:edit",
                "#billing:read",
                "#cache_purge:edit",
                "#dns_records:edit",
                "#dns_records:read",
                "#lb:edit",
                "#lb:read",
                "#legal:edit",
                "#legal:read",
                "#logs:edit",
                "#logs:read",
                "#member:edit",
                "#member:read",
                "#organization:edit",
                "#organization:read",
                "#ssl:edit",
                "#ssl:read",
                "#stream:edit",
                "#stream:read",
                "#subscription:edit",
                "#subscription:read",
                "#teams:edit",
                "#teams:read",
                "#teams:report",
                "#waf:edit",
                "#waf:read",
                "#webhooks:edit",
                "#webhooks:read",
                "#worker:edit",
                "#worker:read",
                "#zone:edit",
                "#zone:read",
                "#zone_settings:edit",
                "#zone_settings:read"
            ],
            "plan": {
                "id": "0feeeeeeeeeeeeeeeeeeeeeeeeeeeeee",
                "name": "Free Website",
                "price": 0,
                "currency": "USD",
                "frequency": "",
                "is_subscribed": true,
                "can_subscribe": false,
                "legacy_id": "free",
                "legacy_discount": false,
                "externally_managed": false
            }
        }
      ]
}